Floria/HW7/HW7.1/3.py

```diff
@@ -0,0 +1,9 @@
+def number_count(str1):
+    dict = {}
```

Comment:
It's a good practice not to use the name dict for variables because it's a built-in type in Python. You can use a more descriptive name like character_count or something similar.